src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/legaladvisor/DocAssemblyTemplateBody.java

```diff
@@ -43,35 +47,35 @@
     @JsonSerialize(using = LocalDateSerializer.class)
     private LocalDate currentDate;
 
-    @JsonProperty("orderDirections1isIncluded")
     private boolean hasFirstOrderDirections;
 
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy")
     @JsonSerialize(using = LocalDateSerializer.class)
     private LocalDate docUploadDeadline;
 
-    @JsonProperty("orderDirections2isIncluded")
     private boolean hasSecondOrderDirections;
 
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy")
     @JsonSerialize(using = LocalDateSerializer.class)
     private LocalDate eyewitnessUploadDeadline;
 
-    @JsonProperty("hearingisRequired")
-    private boolean hearingRequired;
+    @JsonProperty("hearingRequired")
+    private boolean paperDetermination;
 
     private CCDDirectionPartyType docUploadForParty;
 
     private CCDDirectionPartyType eyewitnessUploadForParty;
 
     private String hearingCourtName;
 
-    private String hearingCourtAddress;
+    private CCDAddress hearingCourtAddress;
 
     private CCDHearingDurationType estimatedHearingDuration;
 
     private String hearingStatement;
 
     @JsonTypeInfo(include = JsonTypeInfo.As.WRAPPER_OBJECT, use = JsonTypeInfo.Id.NAME)
-    private List<OtherDirection> otherDirectionList;
+    private List<CCDOrderDirection> otherDirections;
+
+    private List<String> extraDocUploadList;
```

Comment:
can extraDocUploadList be changed to `additionalDocs` pls? List word seems technical. Thanks